test/kafka-exactly-once/mzworkflows.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from typing import List
+
+from materialize.mzcompose import (
+    Kafka,
+    Materialized,
+    SchemaRegistry,
+    Testdrive,
+    Workflow,
+    WorkflowArgumentParser,
+    Zookeeper,
+)
+
+services = [
+    Zookeeper(),
+    Kafka(),
+    SchemaRegistry(),
+    Materialized(),
+    Testdrive(),
+]
+
+
+def workflow_kafka_exactly_once(w: Workflow, args: List[str]):
+    parser = WorkflowArgumentParser(w)
+    parser.add_argument(
+        "--seed",
+        help="an alternate seed to use to avoid clashing with existing topics",
+        type=int,
+        default=1,
+    )
+    args = parser.parse_args(args)
+
+    w.start_services(services=["kafka", "schema-registry", "materialized"])
+    w.wait_for_tcp(host="kafka", port=9092)
+    w.wait_for_tcp(host="schema-registry", port=8081)
+    w.wait_for_mz()
+    w.run_service(
+        service="testdrive-svc",
+        command=f"--seed {args.seed} --kafka-option=group.id=group1 before-restart.td",
```

Comment:
The seed can be supplied directly as an argument to the `Testdrive()` constructor.

Reply:
Unfortunately doing it that way means you can't pass it in as a command-line argument!